error: symbol-name(_RNaCs4fqI2P2rA04_5basic4main)
  --> $DIR/basic.rs:3:1
   |
LL | #[rustc_symbol_name] //~ ERROR symbol-name(_RNaCs4fqI2P2rA04_5basic4main)
   | ^^^^^^^^^^^^^^^^^^^^

error: def-path(main)
  --> $DIR/basic.rs:4:1
   |
LL | #[rustc_def_path] //~ ERROR def-path(main)
   | ^^^^^^^^^^^^^^^^^

error: aborting due to 2 previous errors

