error: symbol-name(_RNpMNdCs4fqI2P2rA04_5impl13fooNnB2_3Foo3bar)
  --> $DIR/impl1.rs:8:9
   |
LL |         #[rustc_symbol_name]
   |         ^^^^^^^^^^^^^^^^^^^^

error: def-path(foo::Foo::bar)
  --> $DIR/impl1.rs:10:9
   |
LL |         #[rustc_def_path] //~ ERROR def-path(foo::Foo::bar)
   |         ^^^^^^^^^^^^^^^^^

error: symbol-name(_RNxMNoCs4fqI2P2rA04_5impl13barNnNdB4_3foo3Foo3baz)
  --> $DIR/impl1.rs:19:9
   |
LL |         #[rustc_symbol_name]
   |         ^^^^^^^^^^^^^^^^^^^^

error: def-path(bar::<impl foo::Foo>::baz)
  --> $DIR/impl1.rs:21:9
   |
LL |         #[rustc_def_path] //~ ERROR def-path(bar::<impl foo::Foo>::baz)
   |         ^^^^^^^^^^^^^^^^^

error: aborting due to 4 previous errors

